### DISCUSSION TITLE
MA HA player plugin

### DISCUSSION THREAD

**Discussion by collateral87** [music-assistant/support#1904] [Q&A] [answered]
Hi,
when i set up the new Homeassistant plugin within MA, nothing happens except of the following popup:

![grafik](https://github.com/music-assistant/hass-music-assistant/assets/97076257/5ff4f531-49c6-43cb-8b58-e12cf0abe379)

is this normal, because this feature isn't ready yet or is this a bug?

**Comment by UsbBitMinter** [comment]
I am also getting the same error :(

**Reply by OzGav** [reply]
After clicking ok can you then fill in the details?

**Reply by collateral87** [reply]
no. the popup close and nothing happens.

**Reply by OzGav** [reply]
Check with beta94. Ensure you clear the browser cache

**Reply by collateral87** [reply]
beta94 is my actual version.
browser cache is cleared and i tried chrome and firefox.
still the same behavior

**Reply by OzGav** [reply]
This is getting looked at

**Comment by chrisghu** [comment, marked as answer]
there is also an open issue for this topic:

[https://github.com/music-assistant/hass-music-assistant/issues/1907](https://github.com/music-assistant/hass-music-assistant/issues/1907)
